tests/acceptance/bootstrap/ArchiverContext.php

```diff
@@ -291,13 +295,23 @@ public function theDownloadedArchiveShouldContainTheseFiles(string $type, TableN
 
 				if ($expectedPath === $actualPath) {
 					if (!$info->getIsdir()) {
+						$fileExtension = \pathinfo($actualPath, PATHINFO_EXTENSION);
 						$fileContent = \file_get_contents("$tempExtractFolder/$actualPath");
-						Assert::assertEquals(
-							$expectedItem['content'],
-							$fileContent,
-							__METHOD__ .
-							" content of '" . $expectedPath . "' not as expected"
-						);
+
+						if (\in_array($fileExtension, ['txt', 'md'])) {
+							Assert::assertEquals(
+								$expectedItem['content'],
+								$fileContent,
+								__METHOD__ . " content of '" . $expectedPath . "' not as expected"
+							);
+						} elseif (\in_array($fileExtension, ['png', 'jpg'])) {
+							Assert::assertFileExists(
+								"$tempExtractFolder/$actualPath",
+								__METHOD__ . " Image file '" . $expectedPath . "' is not in the downloaded archive."
+							);
```

Comment:
this logic doesn't seem future proof, limited extension is tested

Reply:
Looking at resources in `tests/acceptance/filesForUpload`, it doesn't seem practical to check content of all files. For example: `.mp3` audio file.

Can you give some suggestions?